package.json

```diff
@@ -25,6 +25,7 @@
     "postpack": "git tag -a $npm_package_version -m $npm_package_version && git push origin $npm_package_version"
   },
   "dependencies": {
+    "@tastytrade/api": "^5.0.0",
```

Comment:
You removed it from the wronge package.json. You need to remove it from the package.json in the project root. You can keep it in the `examples/package.json`